coc.html

```diff
@@ -1,327 +1,288 @@
 				<li>
-					The use of sexualized language or imagery, and sexual
-					attention or advances of any kind.
-				</li>
-				<li>Repeatedly flouting event rules.</li>
-				<li>
-					The use of derogatory or discriminatory words, sentiments or
-					statements against any race, gender, sexuality and/or
-					religion.
-				</li>
-				<li>
-					Trolling, insulting or derogatory comments, and personal or
-					political attacks.
-				</li>
-				<li>Public or private harassment.</li>
-				<li>
-					Publishing others’ private information, such as a physical
-					or email address, without their explicit permission.
-				</li>
-				<li>
-					Other conduct which could reasonably be considered
-					inappropriate in a professional setting.
+					<a href="https://links.ourtech.community" target="_blank" rel="noreferrer">All Links</a>
 				</li>
 			</ul>
+		</nav>
+	</header>
+
+	<h1>OTC's Code of Conduct</h1>
+
+	<main id="main-content">
+		<h2 id="introduction">Introduction</h2>
+		<p>
+			<a href="https://ourtech.community">Our Tech Community</a>'s
+			Code of Conduct is based on:
+		<ul>
+			<li><a target="_blank" rel="noreferrer"
+					href="https://www.contributor-covenant.org/version/2/1/code_of_conduct">Contributor Covenant
+					(v2.1)</a>
+				and has been modified as allowed by the
+				<a target="_blank" rel="noreferrer" href="https://choosealicense.com/licenses/cc-by-4.0">CC-BY-4.0</a>
+				license.
+			</li>
+			<li>
+				<a target="_blank" rel="noreferrer" href="https://www.recurse.com/social-rules">Recurse Center's Social
+					Rules</a>.
+			</li>
+		</ul>
+		</p>
+
+		<h2 id="our-pledge">Our Pledge</h2>
+		<p>
+			We as members, contributors, and leaders pledge to make
+			participation in our community a harassment-free experience for
+			everyone, regardless of age, body size, visible or invisible
+			disability, ethnicity, sex characteristics, gender identity and
+			expression, level of experience, education, socio-economic
+			status, nationality, personal appearance, race, caste, color,
+			religion, or sexual identity and orientation.
+		</p>
+		<p>
+			We pledge to act and interact in ways that contribute to an
+			open, welcoming, diverse, inclusive, and healthy community.
+		</p>
+
+		<h2 id="our-standards">Our Standards</h2>
+		<p>Examples of <strong>acceptable behavior</strong> include:</p>
+		<ul>
+			<li>Demonstrating empathy and kindness toward other people.</li>
+			<li>Accommodating for people with disabilities.</li>
+			<li>Abide by the rules of the community and event.</li>
+			<li>
+				Being respectful of differing opinions, viewpoints and
+				experiences.
+			</li>
+			<li>Giving and gracefully accepting constructive feedback.</li>
+			<li>
+				Accepting responsibility and apologizing to those affected
+				by our mistakes and learning from the experience.
+			</li>
+			<li>
+				Focusing on what is best not just for us as individuals, but
+				for the overall community.
+			</li>
+		</ul>
```

Comment:
Let's add a `<br/>` here?

![image](https://user-images.githubusercontent.com/61280281/195805030-25f6ea1e-4689-4f47-a74f-9838066542ff.png)

Reply:
This is one of the formatting issues I noticed.  I have a different fix for this prepped. Let's have this PR focus only on adding the rules

Reply:
Alright sure!